src/common/Error/styled.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+
+export const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+`;
+
+export const ErrorImg = styled.img`
+  max-width: 120px;
+`;
```

Comment:
Na figmie jest margines pomiędzy menu a obrazkiem, 180 pikseli. Można dodać jako margines obrazka, ale wtedy na telefonie jest za nisko, więc osobny margines dla wersji mobilnej by trzeba ustawić i osobny dla normalnej.

Reply:
to co zmienić to na 180, a na telefon ile dać?

Reply:
Niee, wielkość obrazka jest w porządku, tylko trzeba dodać margines.

Reply:
Ciężko to stylować bez podpięcia. Może by zmergować i potem się to poprawi?